mozilla-mobile/android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -81,6 +81,12 @@
     <string name="mozac_feature_prompts_nov">Nov</string>
     <!-- December month of the year (short description), used on the month chooser dialog. -->
     <string name="mozac_feature_prompts_dec">Dec</string>
+    <!-- Title of the time picker dialog. -->
+    <string name="mozac_feature_prompts_set_time">Set time</string>
+    <!-- Text used for the millisecond separator in the time picker. -->
+    <string name="mozac_feature_prompts_millisecond_separator">.</string>
```

Comment:
We discussed this string, as well as the following one below, with @flodolo. Even though both strings are localizable, it would be best to use an Android library to define a format of time, that is base on the locale - instead of exposing delimiters.
CC @Alexandru2909

Reply:
Thanks for pinging us! I think the quickest solution we can do here is to move this into a `strings-no-translatable.xml` file like we do with our other components.

Reply:
A fix has been submitted here: https://github.com/mozilla-mobile/android-components/pull/12816

Reply:
As I understand this, it should stay localizable. 
But Android Library has a way to define this, which would be preferable.